src/server/game/Handlers/TradeHandler.cpp

```diff
@@ -237,7 +237,7 @@ static void clearAcceptTradeMode(TradeData* myTrade, TradeData* hisTrade)
     hisTrade->SetInAcceptProcess(false);
 }
 
-static void clearAcceptTradeMode(Item* *myItems, Item* *hisItems)
+static void clearAcceptTradeMode(std::array<Item*, TRADE_SLOT_TRADED_COUNT> myItems, std::array<Item*, TRADE_SLOT_TRADED_COUNT> hisItems)
```

Comment:
This change seems to cause a warning(error) in AppVeyor for line 357:
```cpp
c:\projects\trinitycore\src\server\game\handlers\tradehandler.cpp(357): error C2220: warning treated as error - no 'object' file generated [C:\projects\trinitycore\build\src\server\game\game.vcxproj]
c:\projects\trinitycore\src\server\game\handlers\tradehandler.cpp(357): warning C4700: uninitialized local variable 'hisItems' used [C:\projects\trinitycore\build\src\server\game\game.vcxproj]
c:\projects\trinitycore\src\server\game\handlers\tradehandler.cpp(357): warning C4700: uninitialized local variable 'myItems' used [C:\projects\trinitycore\build\src\server\game\game.vcxproj]
```
https://github.com/TrinityCore/TrinityCore/pull/24142/files#diff-5521b6495c30ba901d8ed225feba9bf7L357
```cpp
        setAcceptTradeMode(my_trade, his_trade, myItems, hisItems);
```

Reply:
Cheers, I'll take a look